testutil-gcloud/src/main/java/io/spine/testing/server/storage/datastore/TestDatastores.java

```diff
@@ -146,6 +149,9 @@ private static Credentials credentialsFrom(Resource serviceAccount) throws IOExc
         return credentials;
     }
 
+    /**
+     * Returns a stub local GCP project ID.
```

Comment:
Please rephrase, so that this piece does not contain so many nouns back-to-back.